### PR TITLE
Feature/toast position

### PR DESCRIPTION
**Pull request by danbucholtz**
Toast now supports bottom, middle, top orientation.

The API docs are updated in our source.  The preview app has the necessary changes but they're commented out for now.

Thanks,
Dan

### CODE REVIEW

File: src/components/toast/toast.ts [hunk 1]
```diff
@@ -86,6 +92,13 @@ export class Toast extends ViewController {
   }
 
   /**
+  * @private
+  */
+  isVaidPosition(position: string) {
```
Comment (biesbjerg) [inline src/components/toast/toast.ts]: Typo

Comment (brandyscarney) [inline src/components/toast/toast.ts]: I fixed it after merging :smile: https://github.com/driftyco/ionic/commit/c67ce8c906cfaebe0e591bd43094342690fdb43c

**Comment by RunningV** [comment]
which version?  `2.0.0-beta.7` still didn't work to `position: top`

**Comment by jgw96** [comment]
@RunningV You can go ahead and try this new feature out by installing the latest Ionic 2 nightly by doing `npm install ionic-angular@nightly`. This lets your see our new features and changes before we do a stable release, which should be right around the corner (: Thanks for using Ionic!

**Comment by juniorlinn** [comment]
Is there a way to stack multiple toast notifications on top of each other?

**Pull request merged**